packages/util/src/bn/min.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright 2017-2019 @polkadot/util authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import BN from 'bn.js';
+
+import assert from '../assert';
+
+/**
+ * @name min
+ * @summary Finds and returns the smallest value in an array of BNs.
+ * @example
+ * <BR>
+ *
+ * ```javascript
+ * import BN from 'bn.js';
+ * import { min } from '@polkadot/util';
+ *
+ * min([new BN(1), new BN(3), new BN(2)]).toString(); // => '1'
+ * ```
+ */
+export default function min (
+  ...items: BN[]
+): BN {
+  assert(
+    items && items.length >= 2,
+    'Must provide two or more BN arguments'
+  );
```

Comment:
This assert is useful, but (totally optional), we could add something like https://stackoverflow.com/questions/49910889/typescript-array-with-minimum-length so that the compiler will also throw an error when `bnMin()` or `bnMin(a)`